### PR TITLE
wikimedia links for licenses

### PR DESCRIPTION
**Pull request by taranais**

### CODE REVIEW

File: samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+https://commons.wikimedia.org/wiki/File:Toaster.jpg
```
Comment (justinormont) [inline samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.txt]: Please cite each as:
```
Donovan Govan, [Toaster](https://commons.wikimedia.org/wiki/File:Toaster.jpg), [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)
```

Which will render as:
> Donovan Govan, [Toaster](https://commons.wikimedia.org/wiki/File:Toaster.jpg), [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)

Comment (justinormont) [inline samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.txt]: Please change to a .md (markdown) file from .txt

File: samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Sapoproductions, [Pizza con huevo](https://commons.wikimedia.org/wiki/File:Pizza_con_huevo.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)  
+Agnieszka Kwiecień, [Spinach pizza](https://commons.wikimedia.org/wiki/File:Spinach_pizza.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Dickbauch, [Kaffekanne db](https://commons.wikimedia.org/wiki/File:Kaffekanne_db.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Skottniss, [Coffeepotsteel](https://commons.wikimedia.org/wiki/File:Coffeepotsteel.jpg), [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en)  
+Kaffekjele, [Kaffekjele, Norsk Folkemuseum, NF.2007-0149AB](https://commons.wikimedia.org/wiki/File:Kaffekjele,_Norsk_Folkemuseum,_NF.2007-0149AB.jpg)
```
Comment (justinormont) [inline samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.md]: Kaffekjele is listed under Creative Commons Attribution-Share Alike 4.0 International

File: samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.txt (hunk 1 above)
Comment (justinormont) [reply samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.txt]: Thanks much for fixing

**Comment by CESARDELATORRE** [comment]
Thanks Inigo! 👍

**Pull request merged**